README.md

```diff
@@ -1,55 +1,49 @@
 <!--
                   
-This source file is part of the TemplatePackage open source project
+This source file is part of the Stanford Spezi open-source project
 
 SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
 
 SPDX-License-Identifier: MIT
              
 -->
 
-# TemplatePackage
+# SpeziNetworking
 
-[![Build and Test](https://github.com/StanfordBDHG/SwiftPackageTemplate/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordBDHG/SwiftPackageTemplate/actions/workflows/build-and-test.yml)
-[![codecov](https://codecov.io/gh/StanfordBDHG/SwiftPackageTemplate/branch/main/graph/badge.svg?token=X7BQYSUKOH)](https://codecov.io/gh/StanfordBDHG/SwiftPackageTemplate)
-[![DOI](https://zenodo.org/badge/573230182.svg)](https://zenodo.org/badge/latestdoi/573230182)
-[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordBDHG%2FSwiftPackageTemplate%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/StanfordBDHG/SwiftPackageTemplate)
-[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordBDHG%2FSwiftPackageTemplate%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/StanfordBDHG/SwiftPackageTemplate)
+[![Build and Test](https://github.com/StanfordSpezi/SpeziNetworking/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordSpezi/SpeziNetworking/actions/workflows/build-and-test.yml)
+[![codecov](https://codecov.io/gh/StanfordSpezi/SpeziNetworking/graph/badge.svg?token=emNLUokqWO)](https://codecov.io/gh/StanfordSpezi/SpeziNetworking)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordSpezi%2FSpeziNetworking%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/StanfordSpezi/SpeziNetworking)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordSpezi%2FSpeziNetworking%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/StanfordSpezi/SpeziNetworking)
 
 
-## How To Use This Template
+A collection of networking-related infrastructure to support Spezi applications.
 
-The template repository contains a template Swift Package, including a continuous integration setup. 
+## Overview
```

Comment:
Would be nice if we can either already link to some documentation (given that we know what the links will be) or create an issue to do this in a separate PR.

Would be good to get a simple list of types + one liner, and link to docs to show what it does here in a bullet point list to give people visiting this package a first glimpse what to expect here.

Reply:
Tracked it as a separate issue. Waiting for SPI to build the documentation 👍